test/gurka/gurka.cc

```diff
@@ -52,21 +52,11 @@ std::vector<midgard::PointLL> to_lls(const nodelayout& nodes,
   return lls;
 }
 
-/**
- * build a valhalla json request body
- *
- * @param location_type  locations or shape
- * @param waypoints      sequence of pointlls representing the locations
- * @param costing        which costing name to use, defaults to auto
- * @param options        overrides parts of the request, supports rapidjson pointer semantics
- * @param stop_type      break, through, via, break_through
- * @return json string
- */
 std::string build_valhalla_request(const std::string& location_type,
                                    const std::vector<midgard::PointLL>& waypoints,
-                                   const std::string& costing = "auto",
-                                   const std::unordered_map<std::string, std::string>& options = {},
-                                   const std::string& stop_type = "break") {
```

Comment:
why remove the doc string and why remove the default parameters? this one isnt obvious to me :smile:

Reply:
oh i didnt scroll down, you moved it to the header

Reply:
look into the header ;) needed to access it (or wanted to to save some lines..)